UIInfoSuite2/i18n/default.json

```diff
@@ -36,5 +32,15 @@
   "ShowRainyDay": "Show rainy days",
   "ShowHarvestPricesInShop": "Show shop harvest prices",
   "ShowWhenNewRecipesAreAvailable": "Show when new recipes are available",
-  "ShowToolUpgradeStatus": "Show tool upgrade status"
+  "ShowToolUpgradeStatus": "Show tool upgrade status",
+  "FeelingLucky": "You're feelin' lucky!!",
```

Comment:
The original language keys should be deleted from every language file, as they are no longer used.